src/index.json

```diff
@@ -1491,6 +1491,61 @@
                 },
                 "sha256Digest": "64f64fbc0689a5283f3f124dc48264d386371fb7a003eacfdc09450c66f5e25d"
             }
+        ],
+        "azconfig": [
```

Comment:
Can we rename this to `azureconfig`? `az azconfig ..` would be a little more confusing for customers.

Reply:
As we discussed offline, this will not be the final name for our service. Currently we still can't finalize the name until public preview in Feb. This pull request is used to try extension onboard process and we will push another change for the naming later. Regarding the customers, they won't be able to use our service as we have to whitelist subscriptions.